web/scripts/dev.js

```diff
@@ -20,90 +23,167 @@ const isSudo = args.includes("sudo")
 
 const { createServer } = require("vite")
 
-;(async () => {
-  const start = performance.now()
+// -- MAIN
+
+let shuttingDown = false
 
+async function startup() {
   const doBuild = await question("Build containers? This can take a while. [y/N] -> ")
-  linebreak()
 
   if (doBuild.trim().toLowerCase() === "y") {
```

Comment:
```suggestion
  if (["y", "yes"].contains(doBuild.trim().toLowerCase()) {
```
maybe?

Reply:
Would be `.includes()` but yeah. Will do in my next PR (I'm working on a TypeScript imports PR)